utils/buildcli/buildcli/xcrun.py

```diff
@@ -0,0 +1,18 @@
+import subprocess
+
+
+def find(tool):
+    """
+    Return the path for the given tool, according to `xcrun --find`.
+    If `xcrun --find` cannot find the tool, return None.
+    """
+    try:
+        # `xcrun --find` prints to stderr when it fails to find the
+        # given tool. We swallow that output with a pipe.
+        out = subprocess.check_output(
+            ['xcrun', '--sdk', 'macosx', '--toolchain', 'default',
```

Comment:
Or, for staging, you could make `build-script` interpret that parameter.  Whatever is easier.